arangodb/tests/test_arangodb.py

```diff
@@ -0,0 +1,28 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from typing import Any, Dict
+
+import pytest
+
+from datadog_checks.arangodb import ArangodbCheck
+from datadog_checks.base.stubs.aggregator import AggregatorStub
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .common import METRICS
+
+
```

Comment:
Can you add tests for the incorrect endpoint/critical service check, correct mode and id tags, and unavailable mode and id tags?